addons/forum-live-preview/addon.json

```diff
@@ -24,21 +24,21 @@
   ],
   "settings": [
     {
-      "name": "Refresh rate",
+      "name": "Refresh delay",
       "id": "rate",
       "type": "select",
       "potentialValues": [
         {
-          "id": "slow",
-          "name": "Slow"
+          "id": "fast",
+          "name": "Short"
         },
         {
           "id": "default",
           "name": "Default"
```

Comment:
Is there a reason this wasn't also changed to normal?

Reply:
It's a different kind of "Default"; all the other options are speeds, except for these ones which are phrased like durations. I'm still thinking about it.

Reply:
I don't think this distinction makes sense - "Default" and "Normal" could both refer to a speed or a duration.